examples/uniswap_v2_usdc_swap.rs

```diff
@@ -0,0 +1,357 @@
+use ethers_contract::BaseContract;
```

Comment:
please use alloy, ethers is deprecated

Reply:
I have very limited Rust experience, so probably I'm missing something. But is the project currently compatibly with alloy? Adding `alloy-primitives` crate causes dozens of compile errors across the project. I can look into rewriting this example but tips on which crates I should use/avoid would be helpful.

Reply:
yes, it is compatible with alloy, it uses alloy internally: https://github.com/bluealloy/revm/blob/1ca3d39f6a9e9778f8eb0fcb74fe529345a531b4/crates/primitives/src/lib.rs

any crate should be fine, but it should use the same version as the crate itself, so currently 0.7: https://github.com/bluealloy/revm/blob/1ca3d39f6a9e9778f8eb0fcb74fe529345a531b4/crates/primitives/Cargo.toml#L20